src/EStimLibrary/Core/SpatialModel/LocalizationData.cs

```diff
@@ -1,5 +1,5 @@
 ﻿namespace EStimLibrary.Core.SpatialModel;
-
+using System.Collections.Generic;
```

Comment:
Style thing: put using statements first, then 2 newlines, then namespace, then 2 new lines, then the class